src/test/java/io/r2dbc/proxy/support/MethodExecutionInfoFormatterTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.r2dbc.proxy.support;
+
+import io.r2dbc.proxy.core.ConnectionInfo;
+import io.r2dbc.proxy.core.MethodExecutionInfo;
+import io.r2dbc.spi.ConnectionFactory;
+import org.junit.jupiter.api.Test;
+import org.springframework.util.ReflectionUtils;
+
+import java.lang.reflect.Method;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author Tadaya Tsuyukubo
+ */
+public class MethodExecutionInfoFormatterTest {
+
+    @Test
+    void withDefault() {
+
+        // String#indexOf(int) method
+        Method method = ReflectionUtils.findMethod(String.class, "indexOf", int.class);
+
+        Long target = 100L;
+
+        ConnectionInfo connectionInfo = mock(ConnectionInfo.class);
+        when(connectionInfo.getConnectionId()).thenReturn("ABC");
+
+        MethodExecutionInfo executionInfo = mock(MethodExecutionInfo.class);
```

Comment:
This isn't going to be a binding requirement, but I'd recommend creating a `test` package with full-fledged Mock types for all (or perhaps most important) interface types.  It's pretty obvious that both users _and_ the project need to create fake versions of these types and rather than requiring users to include a mocking library, and write the behavior, wouldn't it be nicer to say:

```java
MockMethodExecutionInfo executionInfo = MockMethodExecutionInfo.builder()
    .threadId(5L)
    .connectionInfo(MockConnectionInfo.builder()
        .connectionId("ABC")
        .build())
    .executeDuration(Duration.ofMillis(23))
    .method(String.class, "indexOf", int.class)
    .target(100L)
    .build()